website/client/src/components/inventory/stable/index.vue

```diff
@@ -706,24 +706,26 @@ export default {
             const eggKey = specialKey.split('-')[0];
             const potionKey = specialKey.split('-')[1];
 
-            animals.push({
-              key: specialKey,
-              eggKey,
-              potionKey,
-              name: this.content[`${type}Info`][specialKey].text(),
-              isOwned () {
-                return isOwned(type, this, userItems);
-              },
-              mountOwned () {
-                return isOwned('mount', this, userItems);
-              },
-              isAllowedToFeed () {
-                return type === 'pet' && this.isOwned() && !this.mountOwned();
-              },
-              isHatchable () {
-                return false;
-              },
-            });
+            if (this.content[`${type}Info`][specialKey].canFind || isOwned('mount', this, userItems)) {
```

Comment:
I think having the `if` here and not as a function in the animal item like `isOwned` a few lines below means that to get a new pet to show up a full refresh or page change is necessary. Why not add a new method below `canFind` that makes the check and is used with a `v-show` or `v-if` in the template?